src/schema/v2/artwork/artworkContextGrids/index.ts

```diff
@@ -62,12 +63,21 @@ export const ArtworkContextGridType = new GraphQLInterfaceType({
 
 export const ArtworkContextGrids: GraphQLFieldConfig<any, ResolverContext> = {
   type: new GraphQLList(ArtworkContextGridType),
+  args: {
+    includeRelatedArtworks: {
+      type: GraphQLBoolean,
```

Comment:
should this be non-null? (worth double-checking that doesn't break a client not passing it in, but I don't think it will b/c of `defaultValue`)

Reply:
Since this is typed as boolean the graphql server won't even accept `null` as an arg value. Since such a query would be rejected it didn't seem necessary to add the non-null constraint (could add a spec to verify but we'd just be testing the framework 🤷🏽).

Reply:
I agree that making this explicitly non-null wouldn't have a behaviorial difference but  it might be useful for development environments as we use the generated schema to support our type inference:

```
# type: GraphQLBoolean
includeRelatedArtworks: Boolean = true

# type: new GraphQLNonNull(GraphQLBoolean),
includeRelatedArtworks: Boolean! = true
```

Reply:
Yeah interesting, I wonder why the spec reported that expected type as `Boolean!` with a bang

I've added the non-null for good measure in f12500e13da46bada994e5a7333586f31424a7bb!